packages/datagrid/src/components/DataGrid/DataGrid.scss

```diff
@@ -4,7 +4,13 @@ $td-datagrid-skeleton-width: 9.6rem !default;
 $td-datagrid-skeleton-height: 9.6rem !default;
 
 .td-grid {
-	color: tokens.$coral-color-neutral-text;
+	--ag-borders: var(--coral-border-s-solid);
+	--ag-borders-row: var(--coral-border-s-solid);
+	--ag-borders-critical: var(--coral-border-s-solid);
+	--ag-border-color: var(--coral-color-neutral-border-weak);
+	--ag-row-border-color: var(--coral-color-neutral-border-weak);
+	--ag-cell-horizontal-border: var(--coral-border-s-solid) var(--coral-color-neutral-border-weak);
```

Comment:
If you don't use the SCSS variables you'll lose all the default values, making this unable to work without ThemeProvider and it would also break in silence if the tokens change.

Reply:
ok then, I'll quit.